utils/fetchAssets.ts

```diff
@@ -65,8 +92,41 @@ export const fetchAsset = async (
       console.warn(`Asset ${address} not found in the index`)
       return []
     }
-    default:
+    case ASSET_TYPES.UNKNOWN: {
+      // for unknown type we do additional RPC check and run getAsset again
+      console.warn(
+        `Asset ${address} standard is unknown, fallback to RPC check`
+      )
+      const standard = await detectStandard(address)
```

Comment:
there is no try catch in that function nor here - I would suggest to add try catch in the function and return UNDEFINED in case of error.

Reply:
Makes sense, I will adjust. I think I will move move this into `default` case handler. So we can do the RPC check any time indexer has issue